tgg/seqr_scripts/sex_check.py

```diff
@@ -192,7 +191,7 @@ def call_sex(
 
     # Filter to pass variants only (empty set)
     # TODO: Make this an optional argument before moving to gnomad_methods
-    mt = mt.filter_rows(mt.filters.length() == 0, keep=True)
+    mt = mt.filter_rows(~hl.is_defined(mt.filters), keep=True) # NOTE: As of v0.2.102, hail imports empty sets as missing/NaN
```

Comment:
```suggestion
    mt = mt.filter_rows(hl.is_missing(mt.filters))
```
(this seems easier to read)

I also can't suggest this on the lines above, but maybe it makes more sense for the comment to be:
```
# Filter to PASS variants only (variants with empty filter set)
# NOTE: As of v0.2.102, hail imports empty sets as missing/NaN
```